src/sentry/integrations/bitbucket/integration.py

```diff
@@ -43,14 +44,30 @@ def get_client(self):
     def username(self):
         return self.model.name
 
+    def get_repositories(self):
+        repos = self.get_client().get_repos(self.username)['values']
+        data = []
+        for repo in repos:
+            data.append(
+                {
+                    'search_key': repo['name'],
+                    'value': repo['full_name'],
+                    'label': repo['name'],
+                }
+            )
+        return [{'id': repo['uuid'], 'name': repo['name'], 'full_name': repo['full_name']}]
```

Comment:
Wait I'm confused now, do we even use `data`?

Reply:
Woops! So sorry. This file shouldn't be here. Thanks! I have a different pr for bitbucket. I think it was me picking out the wrong commits.